pkgs/applications/networking/sync/lsyncd/default.nix

```diff
@@ -37,7 +41,7 @@ stdenv.mkDerivation rec {
     homepage = "https://github.com/axkibe/lsyncd";
     description = "A utility that synchronizes local directories with remote targets";
     license = licenses.gpl2;
```

Comment:
gpl2Only or gpl2Plus?

Reply:
Uh, no idea! But I haven't touched anything there

Reply:
I still saw it and if we want to remove the old licenses we need to change them at some point.

Reply:
```suggestion
    license = licenses.gpl2Plus;
```

Their source code states this https://github.com/axkibe/lsyncd/blob/master/inotify.c#L4.